### DISCUSSION TITLE
Bmw x1 f48 e samsung s24 ultra non si connette

### DISCUSSION THREAD

**Discussion by silvertip79** [BimmerGestalt/AAIdrive#829] [Q&A]
Salve ho collegato il telefono come da procedura dell'appartamento AAI ma quando arriva a in attesa connessione app si blocca e non va avanti. Posso sentire solo la musica tramite bluetooth. Allego le foto del problema
![Screenshot_20241017_191338_AAIdrive](https://github.com/user-attachments/assets/84edbb68-45ad-4991-8ed4-1e324ba44174)
![20241017_191353](https://github.com/user-attachments/assets/a4a32d56-31a4-40e7-a9c6-7a4549d29bf5)

Grazie

**Comment by hufman** [comment]
AAIdrive requires that MyBMW connects to the car. MyBMW is the app that checks the box in the car. Have you installed MyBMW and granted it permissions to connect to Nearby Devices and run in the background without battery optimizations?
